orga/translations/en.json

```
@@ -286,6 +286,50 @@
         "row-title": "Skill"
       }
     },
+    "students-sco": {
+      "actions": {
+        "download-template": "Download the template",
+        "import-file": {
+          "file-type-separator": " or ",
+          "label": "Import ({types})"
+        },
+        "manage-account": "Manage the account"
+      },
+      "agri-tooltip": "Some information about import:<ul><li>For your students: use the FREGATA export towards Pix and click on the import button.</li><li>For your trainees: Pix imports all trainees at the end of November into your Pix Orga workspace. From December on, you will be able to add trainees from time to time. To do so, consult the Pix Orga documentation and use the template provided.</li></ul>",
+      "connexion-types": {
+        "empty": "\u2013",
+        "none": "None",
+        "email": "Email address",
+        "identifiant": "Username",
+        "mediacentre": "Mediacentre"
+      },
+      "know-more": "Know more",
+      "table": {
+        "column": {
+          "date-of-birth": "Date of birth",
+          "first-name": "First Name",
+          "last-name": "Last Name",
+          "login-method": "Log in method"
```

```suggestion
          "login-method": "Log in method(s)"
```
vu qu'il peut y en avoir plusieurs ?

---

à moins qu'en anglais ça se dise pas 🤷